### PR TITLE
MINOR: Fix compilation failures

### PR DESCRIPTION
**Pull request by rajinisivaram**

### CODE REVIEW

**Comment by rajinisivaram** [comment]
@ijuma Can you review please?

**Comment by ijuma** [comment]
Thanks for fixing this @rajinisivaram, my bad.

File: support-metrics-client/src/main/java/io/confluent/support/metrics/SupportedKafka.java [hunk 1]
```diff
@@ -18,7 +18,8 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import java.util.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.Locale;
```
Comment (ijuma) [inline support-metrics-client/src/main/java/io/confluent/support/metrics/SupportedKafka.java]: Why do we need `java.util.Locale`?

Comment (rajinisivaram) [reply support-metrics-client/src/main/java/io/confluent/support/metrics/SupportedKafka.java]: `System.getProperty("os.name").toLowerCase(Locale.ROOT)`

Comment (ijuma) [reply support-metrics-client/src/main/java/io/confluent/support/metrics/SupportedKafka.java]: Makes sense, thanks.

**Comment by ijuma** [comment]
This should target the 4.0.x branch btw.

**Comment by rajinisivaram** [comment]
Closing to target against 4.0.x since there are other commits in master

**Pull request closed without merging**